apps/files/src/components/LocationPicker/LocationPicker.vue

```diff
@@ -9,14 +9,20 @@
       <oc-button @click.native="leaveLocationPicker(originalLocation)">
         <translate>Cancel</translate>
       </oc-button>
-      <oc-button
-        id="location-picker-btn-confirm"
-        variation="primary"
-        :disabled="!canConfirm"
-        @click.native="confirmAction"
-      >
-        <span v-text="confirmBtnText" />
-      </oc-button>
+      <span v-if="!canConfirm" :uk-tooltip="sameLocationToolTip">
+        <oc-button id="location-picker-btn-confirm" disabled>
+          <span v-text="confirmBtnText" />
+        </oc-button>
+      </span>
+      <span v-else>
```

Comment:
Is the wrapping span needed?

Reply:
uk-tooltip on a disabled button is not working (ie11 :vomiting_face: ) and therefore it needs wrapping spans/divs